src/app/test/db/mongo/util_test.py

```diff
@@ -268,6 +281,41 @@ def test_missing(self, model_mocks):
         assert role.save.called is True
 
 
+class TestCommandBlackList(object):
```

Comment:
If you move the blacklist function into `beer_garden/events/__init__.py` as suggested earlier, then this would similarly move to `tests/events/init_test.py`